source/extensions/config_subscription/grpc/grpc_mux_impl.cc

```diff
@@ -91,7 +91,14 @@ void GrpcMuxImpl::onDynamicContextUpdate(absl::string_view resource_type_url) {
   queueDiscoveryRequest(resource_type_url);
 }
 
-void GrpcMuxImpl::start() { grpc_stream_.establishNewStream(); }
+void GrpcMuxImpl::start() {
+  ASSERT(!started_);
+  if (started_) {
```

Comment:
How do we get into this situation now (where it wasn't possible before)?

Reply:
OK, thanks. Normally not a super fan of code that can't be unit tested (as per the assertion), but in this case it's really trivial and essentially a nop.